src/index.js

```diff
@@ -0,0 +1,53 @@
+import './style.css';
+import dragDropIcon from './images/drag-drop-icon.svg';
+
+const toDoTask = [
+  {
+    description: 'task1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'task2',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'task3',
+    completed: false,
+    index: 2,
+  },
+];
+
```

Comment:
- [ ] Great job creating an array of some simple to-do tasks (array of objects) 👏🏻  However, on page load, the tasks should appear in order of the `index` values for each task. I strongly suggest you set the property index to the value of the new array length (i.e. if you have 5th task to the list, the index of that task should equal 5). HINT thought_balloon you can modify indexes of your code and make sure the first index is 1 and not zero  this way when you create a todo list, the index starts from 1 instead of 0